src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -441,6 +476,7 @@ private void OnScroll(object sender, ScrollEventArgs e)
         private void OnScrollAnimationStarted(object sender, ScrollEventArgs e)
         {
             onAnimation = true;
+            if (isScreenReaderEnabled) Accessibility.Accessibility.ClearCurrentlyHighlightedView();
```

Comment:
If you mve this line into the OnScroll, then it will work as we want. ;-D

Reply:
Thank you for this guide!